bpf/bpf_sock.c

```diff
@@ -235,6 +235,7 @@ sock4_wildcard_lookup_full(struct lb4_key *key __maybe_unused,
 	return NULL;
 }
 
+#ifdef ENABLE_LOCAL_REDIRECT_POLICY
```

Comment:
I believe you don't need this `#ifdef`, its always inlined and you already have an `#ifdef` at the call site. We want to 
keep the amount of `#ifdef`s in the code down if we can so going clang-free is easier.

Reply:
As I thought, clang complains about unused code if I removed those ifdefs - 

```
bpf_sock.c:258:1: error: unused function 'sock4_skip_xlate_if_same_netns' [-Werror,-Wunused-function]
sock4_skip_xlate_if_same_netns(struct bpf_sock_addr *ctx __maybe_unused,
```

Reply:
Reverted to the original changes because of failures.

Reply:
Ah, of-course